### ISSUE TITLE
scrollycoding example does not work with the next15 

### ISSUE THREAD

**Issue by qiushiyan** [code-hike/codehike#487]
I forked the starter project, after upgrading to the latest version of next 15 and `@next/mdx` 15.0.4, the `parseRoot` function in the scrollycoding page returns error

```
[ Server ] Error: Cannot read properties of undefined (reading 'A')
```

Moving `content.md` to a standalone page component renders ok. So I guess it has to do with how codehike processes it.

stackblitz link https://stackblitz.com/edit/github-5wcqn232?file=package.json

Commands I used to upgrade the project

```
npx @next/codemod@canary upgrade latest

npm install @mdx-js/loader@latest @mdx-js/react@latest @next/mdx@latest codehike@latest
```

**Comment by stevef24** [comment]
Yes, Having the same issue here, tried installing the same packages too, didnt work ,I thought it was a issue with turbopack

**Comment by pomber** [comment]
I updated the starter to Next 15. Fixed the problem by renaming `content.md` to `content.mdx`. Not sure why Next.js doesn't like the `md` extension. 

Here's the diff: https://github.com/code-hike/v1-starter/commit/d5d04312af910a06cc30e5dbabfa6e91e1162802
